GameData/KerbalismConfig/Support/ROCapsules.cfg

```diff
@@ -601,26 +1002,47 @@
 {
 	@MODULE[Habitat]
 	{
-		%volume = 8.95
-		%surface = 35
+		%volume = 9	//m^3
```

Comment:
The old value was sourced. The difference is almost insignificant, but do you have a reason for the change?